src/footer/footer.jsx

```diff
@@ -30,17 +30,17 @@ const Footer = () => {
                 </div>
                 <div className='funiro_col furniro_col_second'>
                     <h3 className='links_title'>Links</h3>
-                    {<FooterLinksList elem={footerLinks[0]} />}
+                    {<FooterLinksList link={footerLinks.links} />}
                 </div>
                 <div className='funiro_col furniro_col_second'>
                     <h3 className='links_title'>Help</h3>
-                    {<FooterLinksList elem={footerLinks[1]} />}
+                    {<FooterLinksList link={footerLinks.help} />}
                 </div>
                 <div className='funiro_col furniro_col_third'>
                     <h3 className='links_title'>Newsletter</h3>
                     <form className='footer_form' action=''>
                         <input type='email' className='email_footer' placeholder='Enter Your Email Address' />
-                        <button  className='subscribe' >SUBSCRIBE</button>
+                        <button className='subscribe' >SUBSCRIBE</button>
```

Comment:
```suggestion
                        <button type="submit" className='subscribe' >SUBSCRIBE</button>
```
always specify `type` attribute